src/Progress/Progress.tsx

```diff
@@ -31,52 +31,55 @@ interface IProgressProps {
 
 const {width: screenWidth} = Dimensions.get('screen')
 
-const Progress = (
-  {
-    value = 0,
-    size = 16,
-    borderRadius = 0,
-    filledTrackColor = '#49BE25',
-    backgroundColor = '#E5E5E5',
-  }: IProgressProps,
-  ref?: any,
-) => {
-  const [width, setWidth] = useState(0)
-  const translateX = useRef(new Animated.Value(-screenWidth)).current
-  const toTranslateX = useRef(new Animated.Value(-screenWidth)).current
+const Progress = forwardRef<View, IProgressProps>(
+  (
+    {
+      value = 0,
+      size = 16,
+      borderRadius = 0,
+      filledTrackColor = '#49BE25',
+      backgroundColor = '#E5E5E5',
+    }: IProgressProps,
```

Comment:
We can skip this `IProgressProps` defined here, since we already defined them above (`forwardRef<View, IProgressProps>`)

Reply:
updated